### PR TITLE
Move memcache testing from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by glensc**
Re-created from changes of https://github.com/zf1s/zf1/pull/34

This PR moves Memcache related tests from Travis CI to GitHub Actions.

### CODE REVIEW

File: tests/TestConfiguration.ci.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+/**
+ * Zend_Cache
+ *
+ * TESTS_ZEND_CACHE_MEMCACHED_ENABLED  => memcache extension has to be enabled and
+ *                                        a memcached server has to be available
+ * TESTS_ZEND_CACHE_LIBMEMCACHED_ENABLED => memcached extension has to be enabled and
+ *                                          a memcached server has to be available
+ */
+defined('TESTS_ZEND_CACHE_MEMCACHED_ENABLED') || define('TESTS_ZEND_CACHE_MEMCACHED_ENABLED', true);
+defined('TESTS_ZEND_CACHE_LIBMEMCACHED_ENABLED') || define('TESTS_ZEND_CACHE_LIBMEMCACHED_ENABLED', true);
```
Comment (glensc) [inline tests/TestConfiguration.ci.php:43]: @falkenhawk I don't know how to tests that the memcache tests are now run. Count the difference of `S` in outputs? :)

Comment (glensc) [reply tests/TestConfiguration.ci.php:43]: OK, I made separate Run where memcached is not started, everything failed:
- https://github.com/glensc/php-zf1s/actions/runs/512842524

**Comment by glensc** [comment]
- https://github.com/zf1s/zf1/pull/47/checks?check_run_id=1771581283
`Tests: 14504, Assertions: 58196, Incomplete: 19, Skipped: 587.`

**Comment by glensc** [comment]
Rebased after https://github.com/zf1s/zf1/pull/48

**Comment by glensc** [comment]
@falkenhawk merge time?

**Review by falkenhawk — APPROVED** [review]
💚

**Pull request merged**